doc/source/basicgameplay/hacking.rst

```diff
@@ -107,3 +107,10 @@ starting security, rounded to the nearest integer. To be more precise::
 
 This means that a server's security level will not fall below this
 value if you are trying to weaken() it.
+
+Backdoors
+^^^^^^^^^
+Servers that have been hacked can have backdoors installed. These backdoors
```

Comment:
maybe also mention something about the fact that backdooring a server allows direct connection via `connect`

Reply:
Didn't know that was a thing. I'll add it now.

Reply:
neither did i till the other day 😆